### ISSUE TITLE
Mzmine 2.26 basline-corretion issue 

### ISSUE THREAD

**Issue by MathiuPerez** [mzmine/mzmine2#313]
Hi!

I have noticec that the latest version of MZmine (2.26) does not successfully corrects the baseline as a previous version used to do so (2.21). Please let me know if this is in fact true or if I'm doing anything wrong. 

Thanks!

**Comment by tomas-pluskal** [comment]
Hi,

Not sure, please provide more details so we can debug it.

**Comment by MathiuPerez** [comment]
Hi, 

Thank you for your prompt response.  The issue was when I installed the new version, apparently I missed some packages. I made sure to install all the packages that I needed and the issue was solved. 

Another issue that I just encountered is that, after processing some data as I usually do (has worked before) I'm not obtaining any data after /chronograph builder/. Below, I have attached some images. 

Thanks 


![capture](https://user-images.githubusercontent.com/29205442/26891392-dccfd602-4b82-11e7-9c59-2d2068a2c701.JPG)
